diff --git a/code.cpp b/code.cpp
index a1b8312..e0c83a8 100644
--- a/code.cpp
+++ b/code.cpp
@@ -6,11 +6,11 @@
  *
  * Unauthorized copying of this file, via any medium is strictly prohibited.
  * This file is a proprietary and confidential part of a software product
- * by Umbrella Inc or part of its connected software and documentation.
+ * by Rainbow GbR. or part of its connected software and documentation.
  *
- * ANY SOURCECODE BEARING THIS HEADER SHALL NOT BE RELEASED TO THE
+ * ANY FILE BEARING THIS HEADER SHALL NOT BE RELEASED TO THE
  * PUBLIC, BE USED, BE MODIFIED OR BE DISTRIBUTED IN ANY WAY
- * WITHOUT WRITTEN PERMISSION OF UMBRELLA INC.
+ * WITHOUT WRITTEN PERMISSION OF RAINBOW GBR..
  */
 
 #include <iostream>
