diff --git a/code.cpp b/code.cpp
index 0f068e5..97883a4 100644
--- a/code.cpp
+++ b/code.cpp
@@ -1,7 +1,8 @@
 /*
- * SomeOutdatedFilename.hpp
+ * code.cpp
  *
  * Copyright (c) 2018 Peter Pan
+ * Copyright (c) 2022 Lictools Unittest
  * All rights reserved.
  *
  * Unauthorized copying of this file, via any medium is strictly prohibited.
@@ -19,3 +20,5 @@ int main(int argc, char* argv[]){
     std::cout << "Hello World" << std::endl;
     return 0;
 }
+
+inline void unused(){}
