diff --git a/code.cpp b/code.cpp
index 932efa0..9e5655c 100644
--- a/code.cpp
+++ b/code.cpp
@@ -1,8 +1,17 @@
 /*
  * My Project
  *
- * Copyright (c) 2022 Test Author
+ * Copyright (c) 2022 - 2025 Test Author
  * All rights reserved.
+ *
+ * License: Lorem Ipsum
+ *
+ * Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor
+ * incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+ * exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute
+ * iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+ * pariatur. Excepteur sint obcaecat cupiditat non proident, sunt in culpa qui
+ * officia deserunt mollit anim id est laborum.
  */
 
 #include <iostream>
